### DISCUSSION TITLE
Please help me for sweets collection

### DISCUSSION THREAD

**Discussion by nosfill** [NAIST-SE/AttractingContributors#11] [General]
Hi everyone.

I need help with my sweets list.

Come to [my fork](https://github.com/ae19kitaoka/AttractingContributors-Sweets) and add to my README file.

Add your favorite sweets!

**Comment by vittunyuta-aom** [comment]
added!!

**Reply by nosfill** [reply]
Thank you!

**Comment by wowry** [comment]
Added!

**Reply by nosfill** [reply]
Thank you!

**Comment by satokibi44** [comment]
added!

**Reply by nosfill** [reply]
Very thank you!

**Comment by 278Mt** [comment]
added! and plz help my favorite movie! https://github.com/278mt/AttractingContributors
thx;-)

**Reply by nosfill** [reply]
Thank you!

**Comment by MatsumuraTaiki** [comment]
added! Please help my favorite actors
https://github.com/MatsumuraTaiki/AttractingContributors-Actors

**Reply by nosfill** [reply]
Thank you!
